config/env.production

```diff
@@ -112,6 +112,24 @@ POSTS_URL=https://api.telescope.cdot.systems/v1/posts
 MOCK_REDIS=
 
 
+################################################################################
+# User Service
+################################################################################
+
+# Firebase's private api key. Used to authenticate telescope with firebase
+# See https://firebase.google.com/docs/admin/setup#initialize-sdk to obtain the key
+TYPE=
```

Comment:
Because we have a million env vars, can we please prefix all of these with `FIREBASE_` (e.g., `FIREBASE_PROJECT_ID`)?  It will make it a bit easier to avoid conflicts later.